src/Style/Display/TabsStyle.ts

```diff
@@ -23,35 +37,92 @@ export const TabsHeader = styled.div`
     &::-webkit-scrollbar {
       display: none;
     }
-
-    li {
-      display: inline-flex;    
+    
+    .horizontal-tab.colored {
+      width: 50%;
+      text-align:center;
+      margin: 0;
+    }
+    
+    .horizontal-tab.colored:hover {
+      background-color: ${SecondaryColor.whitesmoke};
+      color: ${SecondaryColor.actionblue};
+      margin: 0;
+    }
+    
+    .horizontal-tab.underlined {
       margin: 0 10px;
+    }
+    
+    .vertical-tab {
+      margin 0 10px;
+      @media (min-width: ${ScreenSize.tablet}px) {
+        margin: 0;
+      }
+    }
+    
+    li {
       text-transform: uppercase;
       list-style-type: none;
-
-      &:first-child {
+      &:first-child
+      .horizontal-tab {
         margin-left: 0;
       }
-
-      &:last-child {
+      
+      &:last-child
+      .horizontal-tab {
         margin-right: 0;
       }
+      
+      &:first-child
+      .vertical-tab {
+        margin-top: 0;
+      }
+
+      &:last-child
+      .vertical-tab {
+        margin-bottom: 0;
+      }
 
-      &.active {
+      &.underlined.active.horizontal-tab,
+      &.active.vertical-tab {
         border-bottom: 2px solid ${SecondaryColor.black};
 
         button {
           font-weight: bold;
           text-transform: uppercase;
         }
       }
+      
+      &.colored.active.horizontal-tab {
+        background-color: ${SecondaryColor.whitesmoke};
+        
+        button {
+          font-weight: bold;
+          text-transform: uppercase;
+        }
+      }
+      
+      &.active.vertical-tab {
+        border-bottom: 2px solid ${SecondaryColor.black};
+        @media (min-width: ${ScreenSize.tablet}px) {
+          border-right: 2px solid ${SecondaryColor.black};
+          border-bottom: none;
+        }
+
+          button {
+            font-weight: bold;
+            text-transform: uppercase;
+          }
+        }
+      }
 
       &:focus {
         outline: none;
       }
 
       button {
+        width: 100%;
```

Comment:
The width of 100% is necessary for the grey(color) themed tabs else there's the UI bug of two tabs getting selected at a time. 
I'll change it to be used only for the colored horizontal tabs

Reply:
Oh hmm my bad. Seems like it's needed for the vertical tabs as well. Now if you click to the right of each tab, the tab won't be selected because the button won't take up the full width. I think you can add `width: 100%` back here, but you also have to add a `text-align: left` for your vertical tab button. Sorry about that!